crates/bevy_math/src/aspect_ratio.rs

```diff
@@ -11,17 +11,60 @@ use bevy_reflect::Reflect;
 pub struct AspectRatio(f32);
 
 impl AspectRatio {
-    /// Create a new `AspectRatio` from a given `width` and `height`.
+    /// Standard 16:9 aspect ratio
+    pub const SIXTEEN_NINE: Self = Self(16.0 / 9.0);
+    /// Standard 4:3 aspect ratio
+    pub const FOUR_THREE: Self = Self(4.0 / 3.0);
+    /// Standard 21:9 ultrawide aspect ratio
+    pub const ULTRAWIDE: Self = Self(21.0 / 9.0);
+
+    /// Create a new ``AspectRatio`` from a given width and height.
```

Comment:
```suggestion
    /// Create a new [`AspectRatio`] from a given width and height.
```
I think you meant square brackets here? Never saw double backwards quotation things, unsure if I just don't know them?

Reply:
You're right, I made a mistake there. I'm sorry for the confusion.